src/pages/Admin/Runtimes/Detail/index.jsx

```diff
@@ -105,6 +107,22 @@ export default class RuntimeDetail extends Component {
         render: item => <TimeShow time={item.status_time} />
       }
     ];
+    const filterList = [
```

Comment:
this block is deplicate, can be moved to one module

Reply:
the file has only one definition about filterList.